.openzeppelin/project.json

```diff
@@ -3,8 +3,8 @@
   "name": "wildcards-contracts",
   "version": "0.1.0",
   "contracts": {
-    "ERC721Patronage": "ERC721Patronage_v1",
-    "WildcardSteward": "WildcardSteward_v2",
+    "ERC721Patronage": "ERC721Patronage_v0",
+    "WildcardSteward": "WildcardSteward_v0",
```

Comment:
Yeah, don't take much note of this, it isn't important. I was just having some issues deploying.